components/src/core/components/Button/Button.vue

```diff
@@ -61,6 +71,14 @@ export default defineComponent({
     iconRightName: {
       type: String,
     },
+    tooltip: {
+      type: String,
+      default: null,
+    },
+    flow: {
+      type: String,
+      default: 'top',
```

Comment:
This kinds of props if validated is better. check [here](https://github.com/orangehrm/oxd/blob/main/components/src/core/components/Input/Select/SelectInput.vue#L119) and [here](https://v3.vuejs.org/guide/component-props.html#prop-validation)

Reply:
fixed

Reply:
validated